### DISCUSSION TITLE
Testing Signals with Angular Testing Library

### DISCUSSION THREAD

**Discussion by giscus** [timdeschryver/timdeschryver.dev#335] [Announcements]
# Testing Signals with Angular Testing Library

TLDR: If you're already using Angular Testing Library, you're in luck, because it's already prepared to handle the Signal properties.

https://timdeschryver.dev/blog/testing-signals-with-angular-testing-library

**Comment by markgoho** [comment]
just for seo purposes it's `jest-preset-angular` and not `jest-angular-preset`

**Reply by timdeschryver** [reply]
I always mix this up 🤦 thanks mentioning it @markgoho !
